install/configuring_inventory_file.adoc

```diff
@@ -550,7 +550,21 @@ the ConfigMap for `node-config-compute` to set `kubeletArguments.pods-per-core`
 to `20`:
```

Comment:
What do you think of adding another example? I'd start at line 547 and say something like this:

```
For example, the following entry in an inventory file defines groups named
`node-config-master`, `node-config-infra`, and `node-config-compute`.
----
openshift_node_groups=[{'name': 'node-config-master', 'labels': ['node-role.kubernetes.io/master=true']}, {'name': 'node-config-infra', 'labels': ['node-role.kubernetes.io/infra=true']}, {'name': 'node-config-compute', 'labels': ['node-role.kubernetes.io/compute=true']}]
----

When you set an entry in the inventory file, you can also edit the ConfigMap for
a node group:

* You can use a list, such as modifying the `node-config-compute` to set `kubeletArguments.pods-per-core` to `20`:
+
----
openshift_node_groups=[{'name': 'node-config-master', 'labels': ['node-role.kubernetes.io/master=true']}, {'name': 'node-config-infra', 'labels': ['node-role.kubernetes.io/infra=true']}, {'name': 'node-config-compute', 'labels': ['node-role.kubernetes.io/compute=true'], 'edits': [{ 'key': 'kubeletArguments.pods-per-core','value': ['20']}]}]
----
* You can use a list to modify multiple key value pairs, such as modifying the
`node-config-compute` group to add two parameters to the `kubelet`:
+
----
openshift_node_groups=[{'name': 'node-config-master', 'labels': ['node-role.kubernetes.io/master=true']}, {'name': 'node-config-infra', 'labels': ['node-role.kubernetes.io/infra=true']}, {'name': 'node-config-compute', 'labels': ['node-role.kubernetes.io/compute=true'], 'edits': [{ 'key': 'kubeletArguments.experimental-allocatable-ignore-eviction','value': ['true']}, {'key': 'kubeletArguments.eviction-hard', 'value': ['memory.available<1Ki']}]}]
----

* You can use also use a dictionary, such as modifying the `node-config-compute`
group to set `perFSGroup` to `512Mi`:
+
----
openshift_node_groups=[{'name': 'node-config-master', 'labels': ['node-role.kubernetes.io/master=true']}, {'name': 'node-config-infra', 'labels': ['node-role.kubernetes.io/infra=true']}, {'name': 'node-config-compute', 'labels': ['node-role.kubernetes.io/compute=true'], 'edits': [{ 'key': 'volumeConfig.localQuota','value': {'perFSGroup':'512Mi'}}]}]
----
```

Reply:
Makes sense, let me rework the PR :+1: